Cesium3DTilesSelection/src/ImplicitQuadtreeLoader.cpp

```diff
@@ -201,7 +221,7 @@ CesiumAsync::Future<TileLoadResult> requestTileContent(
                     logTileLoadResult(pLogger, tileUrl, result.errors);
                     if (result.errors || !result.model) {
                       return TileLoadResult::createFailedResult(
-                          std::move(pAssetAccessor),
+                          pAssetAccessor,
```

Comment:
Moving `pAssetAccessor` in the capture does nothing as `pAssetAccessor` is passed by constant reference.

Reply:
Ah right, so the trick is to put `pAssetAccessor = pAssetAccessor` in the capture list. It doesn't matter that the parameter is passed by const reference, because the capture is by value. But by default (and for no good reason IMO), lambdas capture const things as const even when capturing by value. The silly looking `pAssetAccessor = pAssetAccessor` (which looks like self assignment but it isn't) will work around this problem.